benchmarks/modeling/fitting.py

```diff
@@ -0,0 +1,200 @@
+import numpy as np
+import warnings
+
+from astropy.modeling import models, fitting
+
+
+Gaussian1D = models.Gaussian1D(amplitude=1, mean=0, stddev=1)
+Gaussian2D = models.Gaussian2D(amplitude=1, x_mean=0,  y_mean=0, \
+        x_stddev=1, y_stddev=1)
+
+Polynomial1D = models.Polynomial1D(degree=2)
+Polynomial2D = models.Polynomial2D(degree=2)
+
+Chebyshev1D = models.Chebyshev1D(degree=2)
+Chebyshev2D = models.Chebyshev2D(x_degree=2, y_degree=2)
+
+combined_gauss_1d = models.Gaussian1D(1, 0, 0.1) + models.Gaussian1D(2, 0.5, 0.1)
```

Comment:
Let's have one test with a larger number of models, e.g. 20, instead of 2.

Reply:
How realistic should the parameters of the models being combined be?